operator/internal/controller/vectorized/test/cluster_controller_configuration_test.go

```diff
@@ -429,7 +429,7 @@ var _ = Describe("RedpandaCluster configuration controller", func() {
 		})
 	})
 
-	Context("When setting invalid configuration on the cluster", func() {
+	Context("When setting invalid configuration on the cluster - unknown properties", func() {
```

Comment:
I split this one `Context` into two pieces, in order to narrow down the failure.